src/app/grid/grid-sorting-styling/grid-sorting-styling.component.scss

```diff
@@ -0,0 +1,53 @@
+@import '~igniteui-angular/lib/core/styles/themes/index';
+
+/*BASIC THEMING*/
+
+$custom-theme: igx-grid-theme(
+    $sorted-header-icon-color: blue
+);
+
+/*ENHANCED APPROACHES*/
+
+// $primary-color: #09f;
+// $secondary-color: #e41c77;
+
+/*PALETTE*/
+
+// $light-palette: igx-palette(
+//     $primary: $primary-color,
+//     $secondary: $secondary-color
+// );
+
+/*SCHEMAS*/
+
+// $custom-grid-schema: extend($_light-grid,
+//     (
+//         sorted-header-icon-color: igx-color($light-palette, 'secondary', 500),
+//         // sortable-header-icon-hover-color: blue
+//     )
+// );
+
```

Comment:
Define the schema using the approach in the [**Schemas** topic](https://staging.infragistics.com/products/ignite-ui-angular/angular/components/themes/schemas.html)